lib/_app/AppBar/AppBar.tsx

```diff
@@ -1,61 +1,51 @@
 import {
```

Comment:
I think the AppBar looks good on the homepage

<img width="415" alt="image" src="https://user-images.githubusercontent.com/4027731/165157235-1fa28627-39d3-41f7-99b3-51d6f1518c26.png">

But it doesn't look very good on the celeb page

<img width="414" alt="image" src="https://user-images.githubusercontent.com/4027731/165157330-7673575f-28a4-477b-b477-6f67d39c4603.png">

It is taking more vertical space than necessary.

I propose a few changes:

1. When on the homepage, do not display the subtitle `Important people and facts`. When not on the homepage, display it.
2. When on the homepage, provide a different experience to display the search bar. For example, only show a search button. When the user clicks on that, display the search bar.

But I think it will be difficult to add the logic which checks if we're on the homepage or not. To make your job easier, you can just add a literal `true ? <DoHomePageThings /> : <DoOtherThings />` and then I will figure out how to replace `true` with the correct way to check if we're on the homepage.

Reply:
I'm thinking of removing the logo in the celeb page like this.
![image](https://user-images.githubusercontent.com/68635840/165194235-a91105d3-c972-4231-b4e0-dbcb1e823679.png)

Reply:
I'm not sure this will be beneficial. Most Hollowverse readers land on the website through search engines when they search for things like 'elon musk political views'. They land directly on Elon Musk's page. They would have no idea what website they're on if there's no logo.

Also, redesigning the celeb page is a big undertaking. We'll have to do things in phases. I'm thinking the next phase is to just give the celeb page a facelift, and use that opportunity to remove all of MUI and Sass files from the project.

Reply:
Okay, got it, I will change the wireframe and the App bar in other pages